build.gradle

```diff
@@ -42,7 +43,7 @@ repositories {
 }
 
 allprojects {
-  group 'org.opensearch'
+  group 'org.opensearch.plugin'
```

Comment:
This is how we publish this artifact: https://github.com/opensearch-project/custom-codecs/blob/main/build.gradle#L77